test-requirements.txt

```diff
@@ -0,0 +1,4 @@
+nose
+mock<=1.0.1
```

Comment:
What's the reason for the version restriction?  (I seem to recall an incompatibility with ``six`` in some versions, or something like that)